app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -77,11 +76,11 @@ def get_paginated_records(data, api_request_info):
     return records
 
 
-def get_all_records():
+def get_all_records(fields):
     # Get airtable API URL and add fields to be scraped to URL in HTML format
     headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
     url = AIRTABLE_REQUEST_URL.format(AIRTABLE_BASE_ID)
-    url = add_fields_to_url(url)
+    url = add_fields_to_url(url, fields)
     url += '&filterByFormula={ETL Included}=1'
```

Comment:
Will this `filterByFormula` parameter be needed when we use this function to request for the github CSV?

Reply:
Maybe we could make this function also take in an optional "filter" parameter, which would allow you to append any filter strings onto the url. 

This would make this function super generalizable for all airtable requests (including our normal airtable records request, github csv request, and filter options request)

Reply:
I thought about that and I'm pretty sure we do want this filter by formula to be included because it means it will only show records that haven't been superceded, ones where the denom isn't null, and ones where serum pos prev isn't null which is what we want to show i think. I do like the idea of adding a filter param though so I will go ahead and add that.

Reply:
Hm, I'm actually going to go back on this. We don't use this function for get filter records - it's currently only being used by our normal ETL and by the new github csv script. So right now as it stands, no other pieces of the code use this, and these two applications require the etl included filter. So I can make it a default filters = &filterByFormula={ETL Included}=1' param for now, and then in the future if we need this function it is already generalized. does that make sense?